jujugui/static/gui/src/app/templates/storefront-search-row.partial

```diff
@@ -0,0 +1,113 @@
+<div class="{{#if nameWidth}}{{nameWidth}}{{else}}four-col{{/if}} charm-name__column">
+  <h3 class="list-block__list--item-title">
+    {{#compare docType 'charm'}}
```

Comment:
what is this checking for? I don't see this image in the search results line items in this spot. it's on the right, not left?